tests/integration.rs

```diff
@@ -6728,6 +6748,46 @@ mod common_parallel {
 
         handle_child_output(r, &output);
     }
+
+    #[test]
+    #[cfg(target_arch = "x86_64")]
+    #[cfg(not(feature = "mshv"))]
```

Comment:
Why not? What's stopping this from working on MSHV?

Reply:
done